app/services/proofing/socure/id_plus/request.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+module Proofing
+  module Socure
+    module IdPlus
+      class RequestError < StandardError
+        def initialize(wrapped)
+          @wrapped = wrapped
+          super(build_message)
+        end
+
+        def reference_id
+          return @reference_id if defined?(@reference_id)
+          @reference_id = response_body.is_a?(Hash) ?
+            response_body['referenceId'] :
+            nil
+        end
+
+        def response_body
+          return @response_body if defined?(@response_body)
+          @response_body = if wrapped.respond_to?(:response_body)
+                             wrapped.response_body
+                           end
+        end
+
+        def response_status
+          return @response_status if defined?(@response_status)
+          @response_status = if wrapped.respond_to?(:response_status)
+                               wrapped.response_status
+                             end
+        end
+
+        private
+
+        attr_reader :wrapped
+
+        def build_message
+          message = response_body.is_a?(Hash) ? response_body['msg'] : nil
+          message ||= wrapped.message
+          status = response_status ? " (#{response_status})" : ''
+          [message, status].join('')
+        end
+      end
+
+      class Request
+        attr_reader :config, :input
+
+        SERVICE_NAME = 'socure_id_plus'
+
+        # @param [Proofing::Socure::IdPlus::Config] config
+        # @param [Proofing::Socure::IdPlus::Input] input
+        def initialize(config:, input:)
+          @config = config
+          @input = input
+        end
+
+        def send_request
+          conn = Faraday.new do |f|
+            f.request :instrumentation, name: 'request_metric.faraday'
+            f.response :raise_error
+            f.response :json
+            f.options.timeout = config.timeout
+            f.options.read_timeout = config.timeout
+            f.options.open_timeout = config.timeout
+            f.options.write_timeout = config.timeout
+          end
+
+          Response.new(
+            conn.post(url, body, headers) do |req|
+              req.options.context = { service_name: SERVICE_NAME }
+            end,
+          )
+        rescue Faraday::BadRequestError,
+               Faraday::ConnectionFailed,
+               Faraday::ServerError,
+               Faraday::SSLError,
+               Faraday::TimeoutError,
+               Faraday::UnauthorizedError => e
+
+          if timeout_error?(e)
+            raise ::Proofing::TimeoutError,
+                  'Timed out waiting for verification response'
+          end
+
+          raise RequestError, e
+        end
+
+        def body
+          @body ||= begin
+            {
```

Comment:
the `begin` isn't neede here, just the plain hash literal should be fine
```suggestion
          @body ||= {
```